lib/task.js

```diff
@@ -283,6 +289,8 @@ export class Task {
     this.taskState = 'pending';
     this.options = options;
 
+    this.queue = this.createQueue(this.claim.credentials, runtime);
```

Comment:
I don't think this is necessary yet, no need to keep a queue client around for each reclaimer...jsut recreate it when it's ready to be reclaimed.